components/board/genericlinux/board.go

```diff
@@ -40,7 +41,7 @@ type Config struct {
 }
 
 // RegisterBoard registers a sysfs based board of the given model.
-func RegisterBoard(modelName string, gpioMappings map[int]GPIOBoardMapping) {
+func RegisterBoard(modelName string, gpioMappings map[int]GPIOBoardMapping, useIoctl bool) {
```

Comment:
Eh... not sure the name of this boolean makes sense any more. In the long run, I'd like to get rid of it and have all these boards use the same implementation, but until then, we should have a better name. Maybe usePeriphGpio where every board except the Jetson is set to true for now?